File: est datos/Js/Tarea_2.js
class Libro {
    constructor(titulo, autor) {
        this.titulo = titulo;
        this.autor = autor;
    }

    mostrar() {
        console.log(`Título: ${this.titulo}, Autor: ${this.autor}`);
    }
}

// Crear un arreglo de libros
const biblioteca = [
    new Libro("Cien años de soledad", "Gabriel García Márquez"),
    new Libro("El principito", "Antoine de Saint-Exupéry"),
    new Libro("Berserk", "Kentaro Miura")
];

// Usar los métodos de cada objeto
for (const libro of biblioteca) {
    libro.mostrar();
}

File: est datos/python/Tarea_2.py
class Libro:
    def __init__(self, titulo, autor):
        self.titulo = titulo
        self.autor = autor

    def mostrar(self):
        print(f"Título: {self.titulo}, Autor: {self.autor}")

        
# Crear una lista (arreglo) de libros
biblioteca = [
    Libro("Cien años de soledad", "Gabriel García Márquez"),
    Libro("El principito", "Antoine de Saint-Exupéry"),
    Libro("Berserk", "Kentaro Miura")
]

# Usar los métodos de cada objeto
for libro in biblioteca:
    libro.mostrar()